src/async-components/views/dialogs/secretstorage/CreateSecretStorageDialog.js

```diff
@@ -427,13 +432,57 @@ export default class CreateSecretStorageDialog extends React.PureComponent {
         });
     }
 
+    _renderPhaseChooseKeyPassphrase() {
+        return <form onSubmit={this._onChooseKeyPassphraseFormSubmit}>
+            <p className="mx_CreateSecretStorageDialog_centeredBody">{_t(
+                "Safeguard against losing access to encrypted messages & data by " +
+                "backing up encryption keys on your server.",
+            )}</p>
+            <div className="mx_CreateSecretStorageDialog_primaryContainer" role="radiogroup" onChange={this._onKeyPassphraseChange}>
+                <StyledRadioButton
+                    key={CREATESTORAGE_OPTION_KEY}
+                    value={CREATESTORAGE_OPTION_KEY}
+                    name="keyPassphrase"
+                    checked={this.state.passPhraseKeySelected === CREATESTORAGE_OPTION_KEY}
+                >
+                    <div className="mx_CreateSecretStorageDialog_optionTitle">
+                        <img className="mx_CreateSecretStorageDialog_optionIcon"
+                            src={require("../../../../../res/img/feather-customised/secure-backup.svg")}
+                        />
```

Comment:
hopefully this themes nicely, likewise for other images

Reply:
oh balls :(

Reply:
ok - that should be better